projects/public-search/src/app/api/item.service.ts

```diff
@@ -0,0 +1,120 @@
+/*
+ * RERO ILS UI
+ * Copyright (C) 2021 RERO
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published by
+ * the Free Software Foundation, version 3 of the License.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+import { HttpClient } from '@angular/common/http';
+import { Injectable } from '@angular/core';
+import { Record, RecordService } from '@rero/ng-core';
+import { Observable } from 'rxjs';
+import { map, switchMap } from 'rxjs/operators';
+import { QueryResponse } from '../record.service';
+import { HoldingsService } from './holdings.service';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class ItemService {
+
+  /** Http headers */
+  private _headers = {
+    Accept: 'application/rero+json, application/json'
+  };
+
+  /**
+   * Constructor
+   * @param _recordService - RecordService
+   * @param _httpClient - HttpClient
+   * @param _holdingsService - HoldingsService
+   */
+  constructor(
+    private _recordService: RecordService,
+    private _httpClient: HttpClient,
+    private _holdingsService: HoldingsService
+  ) { }
+
+  /**
+   * Get items by holdings pid and viewcode
+   * @param holdingsPid - string
+   * @param viewcode - string
+   * @param page - number
+   * @param itemsPerPage - number
+   * @return Observable
+   */
+  getItemsByHoldingsPidAndViewcode(
+    holdingsPid: string, viewcode: string, page: number, itemsPerPage: number = 5): Observable<QueryResponse> {
+    return this._recordService
+      .getRecords(
+        'items', `holding.pid:${holdingsPid}`, page, itemsPerPage, undefined,
+        { view: viewcode }, this._headers, 'enumeration_chronology'
```

Comment:
is `enumeration_chronology` sort field is defined into the backend config ?

Reply:
yes, here: https://github.com/rero/rero-ils/pull/1611/files#diff-6be28a31924cc25a9c440ab46eedfa749b3b94ec7a6c5bb3021f76530ac95e4aR2015